lib/ljglibs/gtk/window.moon

```diff
@@ -83,4 +83,14 @@ core.define 'GtkWindow < GtkBin', {
   set_default_icon_from_file: (filename) ->
     catch_error(C.gtk_window_set_default_icon_from_file, filename) != 0
 
+  save_screenshot: (filename, format='png') =>
+    gdkwindow = ffi.C.gtk_widget_get_window ffi.cast 'GtkWidget*', @
+    shot = ffi.C.gdk_pixbuf_get_from_window gdkwindow, 0, 0, @allocated_width, @allocated_height
```

Comment:
Instead of going directly through the FFI we should create a proper wrapper for pixbufs under `ljglibs.Gdk.Pixbuf`. That would then have the `get_from_window` function. This leaks a reference (missing `g_object_unref` handling), and it does not check its return for `NULL`.